src/commands/uwu.ts

```diff
@@ -0,0 +1,46 @@
+import { FormattingPatterns } from 'discord-api-types'
+import { CommandInteraction, TextBasedChannel, TextChannel } from 'discord.js'
+import { Discord, SimpleCommand, SimpleCommandMessage, SlashOption, Slash, SimpleCommandOption } from 'discordx'
+
+
+@Discord()
+class UwU {
+
+  private general_channel = '103678524375699456';
+  private channel: TextChannel | TextBasedChannel | null = null;
```

Comment:
It looks like this is unused so you can just remove it... unless I'm blind which is also possible